docs/src/PolyhedralGeometry/pg_cones.md

```diff
@@ -12,6 +12,36 @@ Pages = ["pg_cones.md"]
 
 # Cones
 
+
 ## Introduction
 
-## Constructions
+Let $\mathbb{F}$ be an ordered field; the most prominent case here is where $\mathbb{F}=\mathbb{Q}$ are the rational numbers.
+
+A set $C \subseteq \mathbb{F}^n$ is called a (polyhedral) cone if it can be written as the set of nonnegative linear combinations of finitely many vectors in $\mathbb{F}^n$.
+Equivalently, cones can be written as the intersection of finitely many homogeneous linear inequalities.
+
+Any cone is a special case of a polyhedron.
```

Comment:
Not part of this review, but: If a cone is a polyhedron, then should perhaps `Cone` be a specialization of `Polyhedron`, or at least some kind of common parent type `AbstractPolyhedron`? (Another subtype then might be `CombinatorialPolyhedron`)